Dockerfile

```diff
@@ -17,6 +17,7 @@ FROM scratch as stage1
 ARG REPOSITORY=https://alpha.de.repo.voidlinux.org
 ARG ARCH=x86_64
 ARG BASEPKG=base-minimal
+ARG ADDINS=spdx-licenses-list
```

Comment:
You can't set this default, you'll break the normal non-masterdir containers.

Reply:
I do not understand what is the problem. Can you elaborate, or hint what i can do?

Reply:
This dockerfile is used to build the masterdirs, as well as minimal containers.  By setting this default, the non-masterdir containers will also all include this package.  You need to either clear the default, or come up with some other way to make sure the containers don't get packages in them they shouldn't have.